agent/engine/docker_task_engine.go

```diff
@@ -301,51 +332,49 @@ func (engine *DockerTaskEngine) synchronizeContainerStatus(container *api.Docker
 			seelog.Warnf("Task engine [%s]: could not find matching container for expected name [%s]: %v",
 				task.Arn, container.DockerName, err)
 		} else {
+			// update the container metadata in case the container status/metadata changed during agent restart
```

Comment:
so if i understand correctly, this codepath is for containers that were ` potentially created while we were down` (debug msg above). is that right? maybe update the comment to reflect that.